streams/src/main/java/org/apache/kafka/streams/kstream/internals/WindowedDeserializer.java

```diff
@@ -64,13 +72,15 @@ public void configure(Map<String, ?> configs, boolean isKey) {
     public Windowed<T> deserialize(String topic, byte[] data) {
 
         byte[] bytes = new byte[data.length - TIMESTAMP_SIZE];
-
+        
+        System.out.println("deserialize method cDs.alled");
```

Comment:
I'm guessing this is left over from debugging, can you take this out?

Reply:
That was the left overs from my attempts, and I probably was sloppy when removing vestiges of my work.